benchmarks/tensorexpr/microbenchmarks.py

```diff
@@ -105,10 +106,10 @@ def gen_float_comparison_tensors(N, M):
 
 te_bool = te.Dtype.Bool
 binary_ops = [
-    ("add", (lambda a, b: a + b), torch.add),
-    ("mul", (lambda a, b: a * b), torch.mul),
-    ("sub", (lambda a, b: a - b), torch.sub),
-    ("div", (lambda a, b: a / b), torch.div),
+    ("add", (operator.add), torch.add),
+    ("mul", (operator.mul), torch.mul),
+    ("sub", (operator.sub), torch.sub),
+    ("div", (operator.truediv), torch.div),
```

Comment:
Nit (I'm surprised we don't have lint for unneeded parentheses)
```suggestion
    ("add", operator.add, torch.add),
    ("mul", operator.mul, torch.mul),
    ("sub", operator.sub, torch.sub),
    ("div", operator.truediv, torch.div),
```